meshtastic/interfaces/ble/receive_service.py

```
@@ -33,82 +34,170 @@
+        if lifecycle_controller is not None:
+            has_ever_connected_session = getattr(
+                lifecycle_controller, "has_ever_connected_session", None
+            )
+            has_ever_connected_session_fn = self._as_usable_callable(
+                has_ever_connected_session
+            )
+            if has_ever_connected_session_fn is not None:
+                try:
+                    result = has_ever_connected_session_fn()
+                except Exception:  # noqa: BLE001 - probe remains best effort
+                    pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)